doc/StaticWorkflow.md

```diff
@@ -0,0 +1,266 @@
+# Preferences Edge Proxy Workflow
+
+A workflow that the Personal Data Storage or Preferences Servier supports
+i where a static site makes save/retrieve requests for the user's preferences.
+An example is where a user changes their UIO options and wants to save them.
+
+Included in this workflow is an OAuth2 authorization sequence where
+users are authenticated by a third party single sign on (SSO) provider, such as
+Google or Github.
```

Comment:
```suggestion
Google or GitHub.
```

Reply:
Thanks for catching all of my typos, and for the suggestions.  I have modified accordingly.